src/simplify_reshapes.cpp

```diff
@@ -185,6 +185,25 @@ struct find_transpose
     }
 };
 
+struct find_literal_convert
```

Comment:
This is a different issue, it would be better to keep it as part of a separate PR

Reply:
All good. I'll remove then